package.json

```diff
@@ -47,6 +47,7 @@
     "react-codemirror2": "^7.2.1",
     "react-color": "^2.19.3",
     "react-confetti": "^6.1.0",
+    "react-copy-to-clipboard": "^5.1.0",
```

Comment:
remove this dependency. implement it with javascript. it need only fewer lines code only.

Reply:
This dependency has been removed.

Reply:
the onclick function to copy text could simply be `() => navigator.clipboard.writeText(shortenedLink);`